config/locales/zh-hk.yml

```diff
@@ -57,226 +57,152 @@ zh-hk:
       updated: 已更新
     schema_name:
       aaib_report:
-        one: 空中意外事故調查報告
```

Comment:
Hey @1pretz1, I just noticed that the `one` value for Chinese is the one we should've been keeping (translated to `Air Accident Investigation Reports`).  The value for `other` was always wrong (it translates to `Other Air Accident Investigation Reports`).  I think what we should've done was swap the value for `one` into the place of `other`, then delete `one`.

This seems to be the case for all Chinese plural forms across all the apps.

Reply:
Below PRs should resolve the issue:
* https://github.com/alphagov/whitehall/pull/7160
* https://github.com/alphagov/collections/pull/3088
* https://github.com/alphagov/government-frontend/pull/2635

Reply:
Thanks!  All approved.